src/Core/src/Core/IView.cs

```diff
@@ -138,5 +139,10 @@ public interface IView : IElement, ITransform
 		/// Method that is called to invalidate the layout of this View.
 		/// </summary>
 		void InvalidateArrange();
+
+#if ANDROID
+		[EditorBrowsable(EditorBrowsableState.Never)]
+		bool Initialized { get; set; }
```

Comment:
I'm concerned about adding a property to this interface only to support a specific platform (and "hiding" it from users).
AFAICT, the purpose of it is to prevent any other property setters from being called before the initial mass property setting during the view/handler setup? If that's the case, I think we could check whether view.Handler is `null` instead. IOW, `UpdateIsEnabled(this AView nativeView, IView view)` would look something like
```
public static void UpdateIsEnabled(this AView nativeView, IView view)
{
	if (view.Handler == null)
		return;
	nativeView.Enabled = view.IsEnabled;
}
```
And even _that_ might not be necessary - the initial barrage of property mappings doesn't happen until the handler/view have been hooked up; I don't think it fires until after `SetVirtualView()` (when `Initialize()` is called). So by the time all these extension methods are being called, the initialization should already be done. 

What might be missing here is some way to tell the PropertyMapper to ignore the properties set by Initialize when it starts up. But that should be an easy change.